manifests/database/postgresql.pp

```diff
@@ -10,6 +10,8 @@
   $db_type                 = $::candlepin::db_type,
   $db_host                 = $::candlepin::db_host,
   $db_port                 = pick($::candlepin::db_port, 5432),
+  $db_ssl                  = $::candlepin::db_ssl
```

Comment:
All class parameters are separated by comma's. init.pp isn't special in terms of syntax.